### DISCUSSION TITLE
Error "No D: reason" (probably if just one year folder)

### DISCUSSION THREAD

**Discussion by apkuki** [TheLastGimbus/GooglePhotosTakeoutHelper#228] [I NEED HELP] [answered]
Hi
I ran into an error -> what I do is: Put all 2021 photos into one google photos album "2021", export that album with Google takeout.

![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/98207992/b283d3d6-dc74-4156-8af3-f22dadbfdb09)

all the takeout .zip are exported in one takeout folder and all of them contain one year "2021" folder.
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/98207992/c74925ea-0af0-45f8-8bbc-72727d2142fe)
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/98207992/bd709981-bb5a-4e62-a052-4eab57989303)

also tried it with cmd directly - did not work either.
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/98207992/73177167-47ab-482e-a39a-5b813b16877d)

**Comment by apkuki** [comment]
i used it the same way like a year ago for 2020 files when it was only cmd. it worked great

**Reply by TheLastGimbus** [reply, marked as answer]
gpth went through a big re-write this year. It now looks for "Photos from XXXX" folders. Export them as shown in my hand-drawn guide in the readme.

**Reply by highmarker** [reply]
Thanks for the tool! It would be great to add that the tool looks for this specific naming convention in the readme. I spent a fair amount of time troubleshooting why I was getting the error before running across this comment.

**Reply by TheLastGimbus** [reply]
Uhh, honestly, detecting all google-photos-stuff folder structure was hard enough ; i don't think detecting other user errors is realistic... that's why i made a graphical tutorial :///

anyway, happy that it works for you now!
